packages/zoe/src/contractFacet/types.js

```diff
@@ -22,7 +22,8 @@
  * synchronously from within the contract, and usually is referred to
  * in code as zcf.
  *
- * @property {Reallocate} reallocate - reallocate amounts among seats
+ * @property {Reallocate} reallocate - reallocate amounts among seats.
+ * Deprecated: Use atomicRearrange instead.
```

Comment:
[@deprecated](https://jsdoc.app/tags-deprecated.html) would get IDEs such as vs-code to highlight uses with strikethru or the like.

(sorry I didn't notice this earlier)

Reply:
but putting it here would tag the `ZCF` type.